src/ui/messages.ts

```diff
@@ -29,9 +29,9 @@ export namespace Messages {
 
   export namespace Dotnet {
     export const IsNotAnExecutableFile = ' is not an executable dotnet file.';
-    export const NotASupportedDotnetInstallation = ' is not a compatible dotnet file. Dafny requires the ASP.NET Core Runtime 5.0 or 6.0, got ';
-    export const FailedDotnetExecution = 'Failed to execute dotnet. Dafny requires the ASP.NET Core Runtime 5.0 or 6.0.';
-    export const NoCompatibleInstallation = 'No compatible dotnet runtime found. Dafny requires the ASP.NET Core Runtime 5.0 or 6.0.';
+    export const NotASupportedDotnetInstallation = ' is not a compatible dotnet file. Dafny requires the .NET Runtime 5.0 or greater, found ';
+    export const FailedDotnetExecution = 'Failed to execute dotnet. Dafny requires the .NET Runtime 5.0 or greater.';
+    export const NoCompatibleInstallation = 'No compatible dotnet runtime found. Dafny requires the .NET Runtime 5.0 or greater.';
```

Comment:
Side note, I'm a bit surprised we need three different error messages, implying there are three different places we check this?

Reply:
Hmm, it's possible that the "Dafny requires the .NET Runtime 5.0 or greater" part is redundant, but indeed the three messages are used in three distinct failure modes (successfully executed `dotnet --list-runtimes` but found no compatible version; failed to execute `dotnet` with a manually-specified executable; and failed to execute `dotnet` using `$PATH`).